ColorizeVolume/ColorizeVolume.py

```diff
@@ -97,13 +109,15 @@ def setup(self) -> None:
         """
         ScriptedLoadableModuleWidget.setup(self)
 
-        if not hasattr(vtkAddon, 'vtkImageLabelDilate3D'):
-            slicer.util.errorDisplay("This module requires a more recent version of Slicer. Please download and install latest Slicer Preview Release.")
+        if not hasattr(vtkAddon, "vtkImageLabelDilate3D"):
+            slicer.util.errorDisplay(
+                "This module requires a more recent version of Slicer. Please download and install latest Slicer Preview Release."
```

Comment:
There is no need for breaking this single line command into 3 lines. It is maybe 10-15 characters narrower, but 3x longer.